developers/contributor-guide/weaviate-core/tests.md

```diff
@@ -23,30 +23,30 @@ This makes tests fast to execute, easy to adapt and easy to run with third-party
+The highest level tests are journey tests. As the top of the pyramid, those tests come with the highest execution cost. There should thus be a few. At the same time they provide a lot of value as they make sure all the components play together. Journey tests don't usually care about edge cases, but rather about validating that a user journey is possible.
 
-Journey-tests are black-box tests. This means the test code is completely unaware of any of the internals of Weaviate. Compare this to the unit or integration tests which tests snippets of (Golang) code. The journey test test an application. The only way for the journey test to interact with the application is through means that are also available to users, such as the public APIs. Our Journey tests are written in Golang to keep the context-switching to a minimum for developers, but the fact that they only test APIs and not code means that they could technically be written in any language.
+Journey-tests are black-box tests. This means the test code is completely unaware of any of the internals of Weaviate. Compare this to the unit or integration tests which tests snippets of (Golang) code. The journey tests test an application. The only way for the journey test to interact with the application is through means that are also available to users, such as the public APIs. Our Journey tests are written in Golang to keep the context-switching to a minimum for developers, but the fact that they only test APIs and not code means that they could technically be written in any language.
 
-This makes sure that the UX for our users is great and the most important features are always working as intended. As a downside they come with the highest execution cost because journey tests need to compile and run the application before being able to run the tests themselves. In addition any runtime [backing service](https://12factor.net/backing-services) must also be present in a test scenario. To make this easy for developers, we provide convenience scripts which build both the application and all backing services and runs them in `Docker Compose`.
+This makes sure that the UX for our users is great and the most important features are always working as intended. As a downside they come with the highest execution cost because journey tests need to compile and run the application before being able to run the tests themselves. In addition any runtime [backing service](https://12factor.net/backing-services) must also be present in a test scenario. To make this easy for developers, we provide convenience scripts which build both the application and all backing services and runs them in `docker compose`.
```

Comment:
This refers to the product ("Docker Compose") rather than the command ("docker compose"). It should be capitalized.